runtime/oti/riscvhelpers.m4

```diff
@@ -28,26 +28,33 @@ define({M},{$2{(}$1{)}})
 
 define({ALen},{8})
 
+dnl
+dnl See J9::RV::PrivateLinkageProperties::PrivateLinkageProperties() in RVPrivateLinkage.cpp
+dnl Following definitions must be kept in sync with private linkage.
```

Comment:
I'm in favour of comments like this but in this case the code/file doesn't exist in this repo at this time so it might be a bit confusing.

Reply:
The code already exists but it's not yet merged: https://github.com/janvrany/openj9/blob/ef6d07d75d65ef28931a6726bcce16318b4b700c/runtime/compiler/riscv/codegen/RVPrivateLinkage.cpp#L52
It will come soon (tm) in subsequent PRs.  I'd prefer to keep the comment, so I won't forget to add it later when chopping my development code into PRs but I'm happy to remove it's a problem.

Thanks!

Reply:
Not a problem, I figured the code existed in your repo. At the very least it's a hint to the reader that a dependency exists so it has some value as it is so let's leave it.